frontend/src/lib/components/AddToDashboard/AddToDashboard.tsx

```diff
@@ -35,7 +28,7 @@ export function AddToDashboard({ insight, canEditInsight }: SaveToDashboardProps
                     </IconWithCount>
                 }
             >
-                Add to dashboard
+                Edit dashboard(s)
```

Comment:
I would say "Edit dashboards" is less clear than "Add to …" in signaling that this is about adding to/removing from dashboards, because it sounds more like editing the dashboards themselves. Maybe "Add to dashboard" if not added anywhere yet and "Manage dashboards" otherwise?